src/dcrypto/v1/v1_test.go

```diff
@@ -92,3 +94,37 @@ func TestRoundTrip(t *testing.T) {
 		}
 	}
 }
+
+func TestHash(t *testing.T) {
+	sizes := []int{24, 1024, 15872, 16364, 16384, 16394, 16896, 66560}
+	for _, size := range sizes {
+		h := md5.New()
```

Comment:
How come in our example our hash func is md5? Could we use another one as IMO it removes a perception that users have of md5. I recognize we are just testing different hashing algos, but some people when reading through code will just look for usage of md5 or sha1 hashers.